boards/friendlyElec-nanoPiM1/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, ... }:
+
+{
+  device = {
+    manufacturer = "FriendlyELEC";
+    name = lib.mkDefault "NanoPi M1";
+    identifier = lib.mkDefault "friendlyElec-nanoPiM1";
+    productPageURL = "http://nanopi.io/nanopi-m1.html";
```

```suggestion
    productPageURL = "https://wiki.friendlyelec.com/wiki/index.php/NanoPi_M1";
```

While the product page is usually preferred, here we'll go for the wiki page, as the product page is from the site previous to the site previous to what is now FriendlyElec. AFAICT it's not sold anymore, so it makes sense to link there instead of the page that is starting to lose functionality.